<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>hanshu_widget</class>
 <widget class="QWidget" name="hanshu_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>746</width>
    <height>656</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/image/big_background.png);</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>761</width>
     <height>661</height>
    </rect>
   </property>
   <widget class="QLabel" name="lab_duishu">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>480</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>250</width>
      <height>200</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/duishu.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>20</y>
      <width>128</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>9</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_14">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="label_one_x1">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text">
          <string>x1:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_one_x1"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_15">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="_x2">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text">
          <string>x2:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_one_x2"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_one_error"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>150</y>
      <width>151</width>
      <height>95</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>9</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_two_a">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text">
          <string>a:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="LE_1y1c_a"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_two_b">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text">
          <string>b:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="LE_1y1c_b"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <item>
        <widget class="QLabel" name="label_two_b_2">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text">
          <string>c:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="LE_1y1c_c"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>500</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_18">
     <item>
      <widget class="QLabel" name="label_five_x">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>x:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_five_x"/>
     </item>
    </layout>
   </widget>
   <widget class="QRadioButton" name="radioButton_1yuan1c">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>160</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_4">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>260</y>
      <width>151</width>
      <height>56</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_three_a">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text">
          <string>a:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="LE_fan_a"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="label_three_b">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text">
          <string>b:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="LE_fan_b"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="lab_fan">
    <property name="geometry">
     <rect>
      <x>109</x>
      <y>260</y>
      <width>201</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>250</width>
      <height>200</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/fan.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_5">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>480</y>
      <width>151</width>
      <height>82</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_five_a">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text">
          <string>a:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="LE_dui_a"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QLabel" name="label_five_b">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text">
          <string>b:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="LE_dui_b"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="lab_1y1c">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>150</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>250</width>
      <height>200</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/1yuan.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_dui">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>500</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lab_zhishu">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>370</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>250</width>
      <height>200</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/zhishu.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Calculate">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>590</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculate</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_6">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>270</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_16">
     <item>
      <widget class="QLabel" name="label_three_x">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>x:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_three_x"/>
     </item>
    </layout>
   </widget>
   <widget class="QRadioButton" name="radioButton_1y2c">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_7">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>380</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_17">
     <item>
      <widget class="QLabel" name="label_four_x">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>x:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_four_x"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_8">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>370</y>
      <width>151</width>
      <height>56</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_four_a">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text">
          <string>a:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="LE_zhi_a"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="label_four_b">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text">
          <string>b:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="LE_zhi_b"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btn_ClearAll">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>590</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>ClearAll</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_9">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>170</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_13">
     <item>
      <widget class="QLabel" name="label_two_x">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>x:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_two_x"/>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="lab_1y2c">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>400</width>
      <height>200</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/1yuan2ci.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_10">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>20</y>
      <width>151</width>
      <height>129</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <property name="spacing">
      <number>9</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_one_a">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text">
          <string>a:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="LE_1y2c_a"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_12">
       <item>
        <widget class="QLabel" name="label_one_b">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text">
          <string>b:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="LE_1y2c_b"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_one_c">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text">
          <string>c:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="LE_1y2c_c"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_19">
       <item>
        <widget class="QLabel" name="label_one_d">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text">
          <string>d:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="LE_1y2c_d"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QRadioButton" name="radioButton_fan">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>280</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_zhi">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>390</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
